deploy/olm-catalog/keycloak-operator/7.0.1/keycloak-operator.v7.0.1.clusterserviceversion.yaml

```
@@ -0,0 +1,242 @@
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: security
+    certified: 'False'
+    containerImage: 'quay.io/keycloak/keycloak-operator:v7.0.1'
+    createdAt: 2019-11-08 00:00:00
+    description: 'An Operator for installing and managing Keycloak'
+    repository: 'https://github.com/keycloak/keycloak-operator'
+    support: Red Hat
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "keycloak.org/v1alpha1",
+          "kind": "Keycloak",
+          "metadata": {
+            "name": "example-keycloak",
+            "labels": {
+              "app": "sso"
+            }
+          },
+          "spec": {
+            "instances": 1,
+            "extensions": [
+              "https://github.com/aerogear/keycloak-metrics-spi/releases/download/1.0.4/keycloak-metrics-spi-1.0.4.jar"
+            ],
+            "externalAccess": {
+              "enabled": true
+            }
+          }
+        },
+        {
+          "apiVersion": "keycloak.org/v1alpha1",
+          "kind": "KeycloakRealm",
+          "metadata": {
+            "name": "example-keycloakrealm"
+          },
+          "spec": {
+            "realm": {
+              "id": "basic",
+              "realm": "basic",
+              "enabled": true,
+              "displayName": "Basic Realm"
+            },
+            "instanceSelector": {
+              "matchLabels": {
+                "app": "sso"
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "keycloak.org/v1alpha1",
+          "kind": "KeycloakBackup",
+          "metadata": {
+            "name": "example-keycloakbackup"
+          }
+        }
+      ]
+  name: keycloak-operator.v7.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: "Represents a Keycloak Instance"
+        displayName: "Keycloak"
+        kind: Keycloak
+        name: keycloaks.keycloak.org
+        version: v1alpha1
+      - description: "Represents a Keycloak Realm"
+        displayName: "KeycloakRealm"
+        kind: KeycloakRealm
+        name: keycloakrealms.keycloak.org
+        version: v1alpha1
+      - description: "Represents a Keycloak Backup"
+        displayName: "KeycloakBackup"
+        kind: KeycloakBackup
+        name: keycloakbackups.keycloak.org
+        version: v1alpha1
+  description: |
+    A Kubernetes Operator based on the Operator SDK for installing and managing Keycloak.
+
+    Keycloak lets you add authentication to applications and secure services with minimum fuss. No need to deal with storing users or authenticating users. It's all available out of the box.
+
+    The operator can deploy and manage Keycloak instances on Kubernetes and OpenShift.
+    The following features are supported:
+
+    * Install Keycloak to a namespace
+    * Import Keycloak Realms from the same or other namespaces
```

```suggestion
    * Import Keycloak Realms
```
Better to change this for now since it is namespace scoped only